### PR TITLE
Feature integration test

### PR DESCRIPTION
**Pull request by RigbeGW**

### CODE REVIEW

File: server/data.py [hunk 1]
```diff
@@ -340,13 +340,15 @@ def _count_gt_for_write(self, variants, data, filters, chips, data_type):
                 het['three_gt_probs'] = gt_probs_het
                 hom_alt['three_gt_probs'] = gt_probs_hom
                 wt_hom['three_gt_probs'] = gt_probs_wt_hom
-                missing['three_gt_probs'] = gt_probs_missing                
+                missing['three_gt_probs'] = gt_probs_missing
+                missing['gt'] = '.|.'
+            else:
+                missing['gt'] = './.'
```
Comment (juhis) [inline server/data.py:346]: In future releases chip data may be phased, i.e. missing genotypes will be .|. like in imputed data - just something to keep in mind

File: server/gt_test.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+            d = d.iloc[9:, ]
+            d['gt'] = [item.split(':')[0] for item in d['gt_']]
+            dat_vcf = d[['FINNGENID', 'gt']]
+            dat_vcf.index = list(dat_vcf['FINNGENID'])
+        
+        # re-order
+        dat_gb = None
+        if dat_gb_full is not None:
+            dat_gb = dat_gb_full[['FINNGENID', 'gt']].copy()
+            dat_vcf = dat_vcf.loc[list(dat_gb.index), :]
```
Comment (juhis) [inline server/gt_test.py]: dat_vcf can be None at this point if the variant is not found in the VCF and this would then break?

**Review by juhis — CHANGES_REQUESTED** [review]
Great to have this test! One small comment

File: server/gt_test.py (hunk 2 above)
Comment (anastasia-shch) [reply server/gt_test.py]: Thank you for your comments! That would indeed break the code. I made the updates accordingly, please take a look when you have time.

**Pull request merged**